packages/api/endpoints/distribution.js

```diff
@@ -81,7 +81,7 @@ function handler(event, context, cb) {
         Key: granuleKey
```

Comment:
The fix for my issue is that on line 61, it's supposed to be `redirect_uri` vs `redirect_url`. This is the error from URS: `I, [2018-05-23T11:28:52.227827 #2083]  INFO -- : [5fd10b78-368e-43b3-ac39-4813ed57fbec] Filter chain halted as :valid_request_params rendered or redirected
E, [2018-05-23T11:28:52.227443 #2083] ERROR -- : [5fd10b78-368e-43b3-ac39-4813ed57fbec] Extra params: ["redirect_url"]
E, [2018-05-23T11:28:52.227361 #2083] ERROR -- : [5fd10b78-368e-43b3-ac39-4813ed57fbec] Missing Params: ["redirect_uri"]`

Reply:
Fixed.